### ISSUE TITLE
Add language translations

### ISSUE THREAD

**Issue by nas5w** [nas5w/100-days-of-code-frontend#6]
Some folks have noted it would be beneficial to have this repository translated to other languages. One contributor has already volunteered to translate the repository into Russian (woohoo!).

If anyone has the time and drive to help out with other languages, please reply below with the proposed language and I'll give you the "go-ahead"... just to make sure two people don't create translations in the same language. Thank you!

**Comment by acamilass** [comment]
I'm interested in translating this to Portuguese-BR :)

**Comment by nas5w** [comment]
@acamilass Awesome! Please go ahead and submit a pull request when you're finished (or a draft pull request as you work).

Thank you!

**Comment by djshabby** [comment]
Hi! Any help with Russian language needed? :)

**Comment by simplefeel** [comment]
@nas5w I'm interested in translating this to chinese

**Comment by nas5w** [comment]
@simplefeel Yes, please do! Thank you!

**Comment by phanlyhuynh** [comment]
@nas5w Hi! I'm interested in translating this to Vietnamese

**Comment by nas5w** [comment]
@phanlyhuynh wow, that's awesome! Please do!

**Comment by JackZeled0n** [comment]
@nas5w Spanish I can!

**Comment by oarevalol** [comment]
@nas5w I'm interested in translating to Spanish :)

**Comment by nas5w** [comment]
@oarevalol please do! Thank you!!

**Comment by Zardosh** [comment]
@nas5w I'm interested in translating to Portuguese-BR. I see someone already said they would like to, but it was a while ago and the translation hasn't been submitted yet, so I'd like to help if possible.

**Comment by nas5w** [comment]
@Zardosh yes, I suspect they have given up. Please do submit!

**Comment by Zardosh** [comment]
@nas5w Would you like me to keep the Translations section or should I remove it?

**Comment by nas5w** [comment]
@Zardosh leave it in please!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by devwolf75** [comment]
I am interested in helping with the spanish translation.

**Comment by nas5w** [comment]
@devwolf75 @oarevalol already claimed the Spanish translation 5 days ago, but perhaps he needs some assistance. Please coordinate with him!

**Comment by rossi-p** [comment]
> @Zardosh yes, I suspect they have given up. Please do submit!

I know you already claimed this, but i would like to help you, if its ok.

**Comment by Zardosh** [comment]
@rossi-p Oh, I have already created a PR with it translated, but I'd appreciate if you could review it and suggest any changes!

**Comment by asyraf-labs** [comment]
@nas5w , I am interested in helping with Malay 🇲🇾 translation. Malay is similar to the Indonesian 🇮🇩 language

**Comment by nas5w** [comment]
@asyraf-labs yes please! Thank you!

**Comment by asyraf-labs** [comment]
Thanks @nas5w , I'm working on it 👨🏻‍💻💪

**Comment by JimmyJoo** [comment]
@nas5w 
Should have known this project earlier! This is a great roadmap to front-end dev. 
I am interested in helping with Korean🇰🇷 translation. 😃

**Comment by meiiwe** [comment]
I'm interested in helping with Indonesian `ID` Translation

**Comment by shoxruxpro** [comment]
@nas5w  I am eager to translate this into Uzbek language :D

**Comment by nas5w** [comment]
@shoxruxpro yes please!!

**Comment by Daniel-Singer** [comment]
Hi! I am keen helping with German 😃

**Comment by nas5w** [comment]
@Daniel-Singer perfect, thank you!

**Comment by miily8310s** [comment]
@nas5w  I'm interested in translating this to japanese :)

**Comment by nas5w** [comment]
@miily8310s I would really appreciate that, thank you!

**Comment by mirajus-salehin** [comment]
I am interested in Bangla translation

**Comment by nas5w** [comment]
@mirajus-salehin yes please!

**Comment by LeonAvancini** [comment]
> @nas5w I'm interested in translating to Spanish :)

Can I help you to translate in spanish too :+1:

**Comment by nas5w** [comment]
@LeonAvancini yes please! That was a while ago so I think the Spanish translation is all yours if you want it.

**Comment by LeonAvancini** [comment]
@nas5w Excelent :).
This will be my first collaboration

**Comment by tselmek** [comment]
Hi, I would like to try my hand at translating it in French ^^

**Comment by nas5w** [comment]
Thank you @tselmek, please do!
